packages/venia-concept/templates/default-font-include.mst

```diff
@@ -0,0 +1,30 @@
+{{!
```

Comment:
The contents of this file are copied from the API response: https://fonts.googleapis.com/css?family=Muli:300

That being said, according to the [docs](https://developers.google.com/fonts/docs/technical_considerations) different browsers solicit a different response from the API which may be a concern.